dev-lang/luau/luau-0.645.ebuild

```diff
@@ -22,17 +22,33 @@ fi
 
 LICENSE="MIT"
 SLOT="0"
-
+IUSE="test"
+RESTRICT="!test? ( test )"
 DOCS=( CONTRIBUTING.md README.md SECURITY.md )
 
 src_test() {
 	"${BUILD_DIR}/Luau.UnitTest" || die
 	"${BUILD_DIR}/Luau.Conformance" || die
 }
 
+src_configure() {
+	local mycmakeargs=(
+		-DLUAU_BUILD_TESTS=$(usex test)
+	)
+	cmake_src_configure
+}
+
 src_install() {
+	dolib.a "${BUILD_DIR}"/libLuau.*.a
+
 	exeinto /usr/bin
 	doexe "${BUILD_DIR}"/luau{,-analyze,-ast,-compile,-reduce}
 
+	insinto /usr/include/Luau
+	doins "${S}"/VM/include/*.h
```

Comment:
No need for `S` at all, I think? if needed , then `cd ${S} || die` and drop S from here